assets/log_example.jsonld

```diff
@@ -0,0 +1,126 @@
+{
+    "@id": "https://github.com/SDSC-ORD/demo_biomedit_workflow/0.1.0/gigantic_lattes/log",
+    "@context": {
+        "owl":    "http://www.w3.org/2002/07/owl#",
+        "prov":   "http://www.w3.org/ns/prov#",
+        "rdfs":   "http://www.w3.org/2000/01/rdf-schema#",
+        "renku":  "https://swissdatasciencecenter.github.io/renku-ontology/",
+        "schema": "http://schema.org/",
+        "xsd":    "http://www.w3.org/2001/XMLSchema#",
+        "gh":     "https://github.com/",
+        "repo":   "https://github.com/SDSC-ORD/demo_biomedit_workflow",
+        "loc":    "file:///home/cmatthey/Repos/github.com/SDSC-ORD/demo_biomedit_workflow/",
+        "@base":  "https://github.com/SDSC-ORD/demo_biomedit_workflow/0.1.0/gigantic_lattes/"
```

Comment:
IIUC, `@base` is the json-ld equivalent of turtle's  `@prefix : <URL>`. It means whenever there is an `@id` without prefix, it will expand to the `@base`. I think the following are all equivalent:

Dedicated context section, can add multiple prefix
```json
{
  "@context": {
      "@base": "http://example.org/"
  }
  "@graph": 
    {
      "@id": "Bob",
      "age": 30
  }
}
```

Simple context, only specify the base by default
```json
{
  "@context": "http://example.org/"
  "@id": "Bob",
  "age": 30
}
```
No context, full URIs
```json

{
  "@id": "http://example.org/Bob",
  "http://example.org/age": 30
}
```

Reply:
This is correct. @base specifies the base IRI .
![image](https://user-images.githubusercontent.com/77491494/226343832-fc053cf1-3f61-4eaf-8adc-c0610a0a9794.png)